presto-native-execution/pom.xml

```diff
@@ -95,6 +95,10 @@
                     <groupId>org.roaringbitmap</groupId>
                     <artifactId>RoaringBitmap</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Any reason why we are excluding these from this version? Since they were not excluded from any older Hudi version? The linux-presto-e2e-tests failures are probably related.

Reply:
oh i excluded that because `TestPrestoSparkNativeJoinQueries` in presto-native-execution was failing due to `java.lang.NoClassDefFoundError: org/objenesis/strategy/InstantiotorStrategy`. After comparing the dependency tree of presto-native-execution for this patch vs master, the only additional thing was `kryo-shaded` coming from hudi-presto-bundle. I thought tests in presto-native execution won't need hudi-presto-bundle so I excluded it. Let me replace this exclusion by 
```
<exclusion>
    <groupId>com.esotericsoftware</groupId>
    <artifactId>kryo-shaded</artifactId>
</exclusion>
```
instead of whole bundle.